tests/MongoDB.Driver.Tests/ClusterKeyTests.cs

```diff
@@ -14,12 +14,14 @@
 */
 
 using System;
+using System.Collections.Generic;
```

Comment:
There are a number of namespaces(added in previous tickets) which are not used in this class. They can be removed if you don't mind

Reply:
Done.